src/MSALWrapper/AuthFlow/IntegratedWindowsAuthentication.cs

```diff
@@ -72,7 +72,8 @@ protected override async Task<TokenResult> GetTokenInnerAsync()
             }
             catch (MsalClientException ex) when (ex.Message.Contains("WS-Trust endpoint not found"))
             {
-                this.logger.LogWarning($"IWA only works on Corp Net, please turn on VPN.");
+                this.logger.LogWarning($"IWA only works on corporate AD backed network, AzureAuth is trying to use other auth flows.");
+                this.logger.LogWarning($"If you specify the IWA mode, please turn on VPN.");

```

Comment:
Should we instead say, "Turn on VPN for IWA mode to succeed". The user might not have externally specified this mode and it could lead to confusion?